packages/components/psammead-most-read/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.2.0 | [PR#3120](https://github.com/bbc/psammead/pull/3120) Remove `MostRead` and `MostReadSection` as well as other cleanup |
```

Comment:
This should be a major version update `2.0.0` since it's changing backward compatibility by removing exports.